tests/components/onkyo/test_config_flow.py

```diff
@@ -198,6 +199,114 @@ async def mock_discover(discovery_callback, timeout):
     assert select_result["description_placeholders"]["name"] == "type 42 (host 42)"
 
 
+async def test_ssdp_discovery_success(
+    hass: HomeAssistant, default_mock_discovery
+) -> None:
+    """Test SSDP discovery with valid host."""
+    discovery_info = ssdp.SsdpServiceInfo(
+        ssdp_location="http://192.168.1.100:8080",
+        upnp={ssdp.ATTR_UPNP_FRIENDLY_NAME: "Onkyo Receiver"},
+        ssdp_usn="uuid:mock_usn",
+        ssdp_udn="uuid:00000000-0000-0000-0000-000000000000",
+        ssdp_st="mock_st",
+    )
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_SSDP},
+        data=discovery_info,
+    )
+
+    assert result["type"] == FlowResultType.FORM
+    assert result["step_id"] == "configure_receiver"
+
+
+async def test_ssdp_discovery_host_info_error(hass: HomeAssistant) -> None:
+    """Test SSDP discovery with host info error."""
+    discovery_info = ssdp.SsdpServiceInfo(
+        ssdp_location="http://192.168.1.100:8080",
+        upnp={ssdp.ATTR_UPNP_FRIENDLY_NAME: "Onkyo Receiver"},
+        ssdp_usn="uuid:mock_usn",
+        ssdp_st="mock_st",
+    )
+
+    with patch(
+        "homeassistant.components.onkyo.receiver.pyeiscp.Connection.discover",
+        side_effect=OSError,
+    ):
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": config_entries.SOURCE_SSDP},
+            data=discovery_info,
+        )
+
+    assert result["type"] == FlowResultType.ABORT
+    assert result["reason"] == "unknown"
+
+
+async def test_ssdp_discovery_host_none_info(
+    hass: HomeAssistant, stub_mock_discovery
+) -> None:
+    """Test SSDP discovery with host info error."""
+    discovery_info = ssdp.SsdpServiceInfo(
+        ssdp_location="http://192.168.1.100:8080",
+        upnp={ssdp.ATTR_UPNP_FRIENDLY_NAME: "Onkyo Receiver"},
+        ssdp_usn="uuid:mock_usn",
+        ssdp_st="mock_st",
+    )
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_SSDP},
+        data=discovery_info,
+    )
+
+    assert result["type"] == FlowResultType.ABORT
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.ABORT
```

`FlowResultType` Enum members are singletons so we should use `is`